mmv1/products/datafusion/api.yaml

```diff
@@ -103,6 +103,10 @@ objects:
         name: 'enableStackdriverMonitoring'
         description: |
           Option to enable Stackdriver Monitoring.
+      - !ruby/object:Api::Type::Boolean
```

Comment:
Do we know if the field can be updated in place? If not, we need to add `input: true`. More details can be found at https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1#apiyaml

Reply:
The behavior of this field is quite different. Hence, will need to have a look and check if that is the case, if so I'll add the necessary changes.

Reply:
I've tried to update the field and the apply passes but the value doesn't get updated. So, the patch call passes the field with new value and in the UI it also says Updating but when we check the value returned (state file), it is unchanged. I've discussed this with Stephen and he suggested to take your inputs on this. Any thoughts?

Reply:
yeah, if you tested it and confirmed that the field was't able to be updated, then adding `input: true` sounds good.

Reply:
Thanks, I've added `input:true` to this field.